docker/Dockerfile-prod

```diff
@@ -11,6 +11,8 @@ CMD ["/usr/local/bin/uwsgi"]
 
 RUN addgroup -g 10001 app && adduser -D -u 10001 -G app -h /app app
 
+ENV PYTHONUNBUFFERED=1
```

Comment:
we don't need this in production. In production we will be using mozlog which doesn't have this problem. (And I'm unsure what this will do in production, do you think it is fine to have in prod??)